server/pulp/plugins/util/nectar_config.py

```diff
@@ -51,7 +40,7 @@ def importer_config_to_nectar_config(importer_config):
         (constants.KEY_MAX_SPEED, 'max_speed'),
     )
 
-    download_config_kwargs = {}
+    download_config_kwargs = {'working_dir': common_utils.get_working_directory()}
```

Comment:
:+1:  setting the working_dir here.

You'll need to do a follow up PR to update the nectar requirement on nectar in the platform spec file once the nectar change has been packaged and you know the version.